.gitignore

```diff
@@ -19,3 +19,18 @@ bundle.json
 *~
 Package.resolved
 extensions/sample/generated
+
+plugins/gnostic-go-generator/examples/v3.0/bookstore/service/service
```

Comment:
Was there an issue with these? I think we should keep these updated as long as they are checked in.